<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Question1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>350</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="text">
    <string>Security Question1：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Answer2content">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>590</y>
     <width>371</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 80);</string>
   </property>
  </widget>
  <widget class="QLabel" name="PasswordConfirm">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="text">
    <string>Confirm it：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButtonCancel">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>630</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background:rgb(245,249,252);
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QPushButton:pressed
{
    background:rgb(245,249,252);
    border-radius:2px;
	padding-left:2px;
	padding-top:2px;
    border: 1px solid rgb(173 , 173 , 173);
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Pic.qrc">
     <normaloff>:/new/prefix1/Pic/a3t.png</normaloff>:/new/prefix1/Pic/a3t.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>700</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>700</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>700</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix1/Background/CHAT.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="Answer1content">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>450</y>
     <width>371</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 80);</string>
   </property>
  </widget>
  <widget class="QLabel" name="Question2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>490</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="text">
    <string>Security Question2：</string>
   </property>
  </widget>
  <widget class="QLabel" name="Designedby">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>670</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="text">
    <string>Designed by Young Inc © 2021 </string>
   </property>
  </widget>
  <widget class="QLabel" name="Answer1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>420</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="text">
    <string>Answer of Security Question1：</string>
   </property>
  </widget>
  <widget class="QLabel" name="Answer2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>560</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="text">
    <string>Answer of Security Question2：</string>
   </property>
  </widget>
  <widget class="QLabel" name="Username">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="text">
    <string>Please select a user name：</string>
   </property>
  </widget>
  <widget class="QLabel" name="Username_2">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>120</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="text">
    <string>Please choose your profile photo：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Question1Content">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>371</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 80);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Question2Content">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>520</y>
     <width>371</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 80);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 80);</string>
   </property>
  </widget>
  <widget class="QLabel" name="Password">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="text">
    <string>Code：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PasswordConfirmEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>310</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 80);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Acount">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255, 80);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pButtonOk">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>630</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
    background:rgb(245,249,252);
    border-radius:2px;
    border: 1px solid rgb(173 , 173 , 173);
}
QPushButton:pressed
{
    background:rgb(245,249,252);
    border-radius:2px;
	padding-left:2px;
	padding-top:2px;
    border: 1px solid rgb(173 , 173 , 173);
}</string>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QLabel" name="Username_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>590</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>Question1</zorder>
  <zorder>Answer2content</zorder>
  <zorder>PasswordConfirm</zorder>
  <zorder>pButtonCancel</zorder>
  <zorder>toolButton</zorder>
  <zorder>Answer1content</zorder>
  <zorder>Question2</zorder>
  <zorder>Designedby</zorder>
  <zorder>Answer1</zorder>
  <zorder>Answer2</zorder>
  <zorder>Username</zorder>
  <zorder>Username_2</zorder>
  <zorder>Question1Content</zorder>
  <zorder>Question2Content</zorder>
  <zorder>PasswordEdit</zorder>
  <zorder>Password</zorder>
  <zorder>PasswordConfirmEdit</zorder>
  <zorder>Acount</zorder>
  <zorder>pButtonOk</zorder>
  <zorder>Username_3</zorder>
 </widget>
 <resources>
  <include location="Pic.qrc"/>
 </resources>
 <connections/>
</ui>
